docs/rfcs/008-more-access-control-oidc-oauth.md

```diff
@@ -0,0 +1,95 @@
+# Open Id Connect / OAuth
```

Comment:
OpenID Connect / OAuth 2.0*

FWIW, this is an OIDC specific implementation since it depends explicitly on ID tokens (which exist as part of the OIDC specification only). Any references to OAuth 2.0 can probably be replaced with references to OIDC.

Reply:
Updated usages.